galaxy_importer/collection.py

```diff
@@ -137,11 +176,31 @@ def __init__(self, path, filename, cfg=None, logger=None):
 
         self.content_objs = None
         self.metadata = None
+        self.file_manifest_file = None
         self.docs_blob = None
         self.contents = None
 
     def load(self):
-        self._load_collection_manifest()
+        # NOTE: If we knew the chksum for MANIFEST.json, we could check it here first
+        self.collection_manifest = self._load_collection_manifest()
+
+        self.metadata = self.collection_manifest.collection_info
+
+        # The default name for 'file_manifest_file' is FILES.json
+        self.file_manifest_file = self.collection_manifest.file_manifest_file
+
+        # check the FILES.json file's chksum
+        check_artifact_file(path_prefix=self.path, artifact_file=self.file_manifest_file)
+
+        # load data from FILES.json
+        self.file_manifest = \
+            self._load_collection_file_manifest(path_prefix=self.path,
```

Comment:
this is referred to as `file_manifest` elsewhere, maybe s/_load_collection_file_manifest/_load_file_manifest

Reply:
Yeah, I'm inconsistent here. 

one extreme

``` python
# since this is returning a CollectionArtifactFileManifest.
#  But it is kind of redundant in a CollectionLoader() class...
collection_artifact_file_manifest =  self._load_collection_artifact_file_manifest(...)
```

given context other way
``` python
# Being in CollectionLoader partially implies we are reading a CollectionArtifact artifact file (tarball)
file_manifest = self._load_file_manifest(...)
```

Though, then I am inconsistent with

``` python
self.collection_manifest = self._load_collection_manifest()
```

I think I'm incline to leave it as is at the moment.

Reply:
Could ditch the 'collection' from all here...

``` python
self.manifest = self._load_manifest()
self.file_manifest = self._load_file_manifest()
```

Reply:
I picked https://github.com/ansible/galaxy-importer/pull/121#discussion_r647636656
updated

Reply:
> Could ditch the 'collection' from all here...
> 
> ```python
> self.manifest = self._load_manifest()
> self.file_manifest = self._load_file_manifest()
> ```

that seems like it would work